### DISCUSSION TITLE
Scenarios: add ability to specify session for the steps

### DISCUSSION THREAD

**Discussion by Zaytsev11B** [yaalalabs/fixyl#20] [Ideas]
For deterministic scenarios, where expected outcome depends on the market data on engine or if testing OXO (taker vs maker), it would be usefult to be able to specify the session for input/output messages, e.g.:

Send Limit DAY to buy @X in session 1
Send Limit DAY to sell @X in session 2
Check fill rate and quantity in session 1 and session 2

**Comment by DhananjayaR** [comment]
Appreciate your suggestion. Will try to implement this feature in a future release and update you here. Thanks again for showing interest in this tool.
